### PR TITLE
expand stage variables

### PR DESCRIPTION
**Pull request by big-kahuna-burger**

### CODE REVIEW

File: modules/util.js [hunk 1]
```diff
@@ -1,3 +1,11 @@
-exports.getTokenOfEvent = function(event){
+exports.getTokenOfEvent = function(event) {
     return event.params.header.Authorization.split(' ')[1];
-}
\ No newline at end of file
+};
+
+exports.extractStageVariables = function (event) {
```
Comment (amirsabersharifi) [inline modules/util.js]: Would you write a mocha test for this function?

Comment (big-kahuna-burger) [inline modules/util.js]: sure

**Comment by big-kahuna-burger** [comment]
I got rid of that access of single element in array using transform function.
https://github.com/request/request-promise#the-transform-function

File: modules/vendor.js [hunk 2]
```diff
@@ -97,14 +97,18 @@ exports.deleteForceVendor = function(vendorId) {
...
+    var options = {
         uri: postgrestUrl + '/Vendors?id=eq.' + vendorId,
         method: 'PATCH',
         json: true,
         body: vendorProperties,
         resolveWithFullResponse: true,
         headers: {
             'Prefer': 'return=representation'
+        },
+        transform: function (body, response, resolveWithFullResponse) {
```
Comment (amirsabersharifi) [inline modules/vendor.js]: I like the idea of transform function, but I believe we can add this function to util and just use it here. What so you think?

**Pull request merged**